probe/WORKSPACE

```diff
@@ -4,7 +4,7 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl","git_repository")
 git_repository(
     name = "px",
     remote = "https://github.com/KayzzzZ/pixie.git",
-    commit = "6b853b96987e570e0979620537c23df5850c7b76",
+    commit = "2d45f14dc69c060260c5634b7718f05f3f0c7bdb",
```

Comment:
Please change repo to kindling-project/pixie

Reply:
ok ...